pkgs/games/frozen-bubble/default.nix

```diff
@@ -16,6 +16,27 @@ perlPackages.buildPerlModule {
   propagatedBuildInputs = with perlPackages; [ AlienSDL CompressBzip2 FileShareDir FileWhich IPCSystemSimple LocaleMaketextLexicon ];
 
   perlPreHook = "export LD=$CC";
+  
```

Comment:
Trailing whitespace (I think? Github doesn't show it, but my local git does.)

Reply:
Good catch! Fixed.